source/utilities.tex

```diff
@@ -5170,12 +5170,18 @@
 
 \indexlibrary{\idxcode{any}!constructor}%
 \begin{itemdecl}
-template <class T, class... Args>
-  explicit any(in_place_type_t<T>, Args&&... args);
+template <class ValueType, class... Args>
+  explicit any(in_place_type_t<ValueType>, Args&&... args);
 \end{itemdecl}
 
 \begin{itemdescr}
 \pnum
+Let \tcode{T} be equal to \tcode{decay_t<ValueType>}.
```

Comment:
I would leave out "equal to" - it's not in the paper, it's inconsistent with other "Let"s, and it adds nothing.